source/codegen/common_helpers.py

```diff
@@ -609,6 +609,18 @@ def is_init_method(function_data):
     return function_data.get('init_method', False)
 
 
+def _get_session_output_param(function_data):
```

Comment:
Do we want to keep the changes in source/codegen since we went with the custom implementation and there aren't any generated changes? I think the methods and updates look fine, was just wondering if we still want/need them.